### PR TITLE
Fix avatars loading below the ground bug

### PR DESCRIPTION
**Pull request by luiscuenca**
This PR fixes the bug that occurs when Address Manager provides a new location for the avatar but the skeleton is not loaded jet. This resulted in a faulty final location due to the bad estimation of the bottom of the avatar capsule (feet).

https://highfidelity.manuscript.com/f/cases/17819/Avatars-are-loading-in-below-the-ground-intermittently

### CODE REVIEW

**Comment by luiscuenca** [comment]
## TEST PLAN
- Wear an avatar with a complex mesh and textures so it takes a while to load

- Go to thespot-dev and situate your avatar by the tables on the first floor and near the elevator.

- Close interface and delete the folders data8 and ktx_cache inside C:\UsersUSERNAME\AppData\Local\High Fidelity\Interface\

- Restart interface. Now you should appear in the middle of the floor.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13340/HighFidelity-Beta-PR14084-82b08f1.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13340/HighFidelity-Beta-PR14084-82b08f1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13340/HighFidelity-Beta-Interface-PR14084-82b08f1.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/13342/HighFidelity-Beta-PR14084-754653b.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13342/HighFidelity-Beta-PR14084-754653b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/13342/HighFidelity-Beta-Interface-PR14084-754653b.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13342/HighFidelity-Beta-PR14084-754653b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/13342/HighFidelity-Beta-Interface-PR14084-754653b.exe

File: interface/src/avatar/MyAvatar.cpp [hunk 1]
```diff
@@ -498,7 +499,7 @@ void MyAvatar::update(float deltaTime) {
     setCurrentStandingHeight(computeStandingHeightMode(getControllerPoseInAvatarFrame(controller::Action::HEAD)));
     setAverageHeadRotation(computeAverageHeadRotation(getControllerPoseInAvatarFrame(controller::Action::HEAD)));
 
-   if (_drawAverageFacingEnabled) {
+    if (_drawAverageFacingEnabled) {
```
Comment (ZappoMan) [inline interface/src/avatar/MyAvatar.cpp]: this whitespace looks weird... is it correct?

Comment (luiscuenca) [reply interface/src/avatar/MyAvatar.cpp]: Yeah, that line was missing a space
https://github.com/highfidelity/hifi/blob/master/interface/src/avatar/MyAvatar.cpp#L501

**Comment by jakram** [comment]
@conklin94122 PASS

Teleporting is working as expected. An observer is able to see the "flight path" of a teleporting avatar. 

Other items tested:

- Multiple domains
- Teleporting through barriers
- Using the teleporter pointed at myself
- Trying to teleport into non-teleporting areas

NOTE: This PR has not been dev approved.

**Comment by qlilaseguinolaza** [comment]
Currently testing

**Pull request merged**